src/components/MerchantsPage/gam/styles.module.scss

```diff
@@ -176,3 +183,115 @@
     font-size: 16px;
   }
 }
+
+.noCampaign {
+  align-items: center;
+  border: 1px solid #e5e5e5;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  padding: 30px;
+
+  @media (max-width: 900px) {
+    grid-template-columns: 1fr;
+    margin-bottom: 15px;
+
+    & > div {
+      margin-bottom: 20px;
+    }
```

Comment:
I prefer not to have selectors like this, because then you tie yourself a rigid nesting structure, and people can accidentally break your code by adding an extra div or changing this div to a span. Instead I'd recommend just creating a separate class for this div

Reply:
Yes, this is a really good point! Will fix.